components/use-cases/ecommerce/ecommerce.md

```diff
@@ -0,0 +1,252 @@
+
+This page contains information about the template. Click **+ Use Template** again.
+
+![Add New Use Case - Shopify to Keboola DWH](/components/use-cases/ecommerce/add-shopify-to-keboola-dwh.png)
+
+Now enter a name for the template instance that you are about to create. This way you can use the template as many times as you want. It is important to keep things organized. 
+
+![Shopify to Keboola DWH - Template Name](/components/use-cases/ecommerce/shopify-to-keboola-dwh-name.png)
+
+After clicking :arrow_right: **Next Step**, you will see the template builder. Fill in all needed credentials and perform the required OAuth authorizations. 
+
+**Important:** Make sure to follow all the steps very carefully to prevent the newly created flow failing because of any user authorization problems. 
+If you are struggling with this part, go to the section [Authorizing Destinations](/components/use-cases/ecommerce/authorizing-destinations/) below.
+
+Follow the steps one by one and authorize your data sources. An Ecommerce data source is required. In this case, it is Shopify.
+
+> Using additional marketing sources is optional. You can (but you do not have to) enrich your eCommerce data with marketing statistics like costs, impressions, and clicks.
+
+Finally, the destination must be authorized as well. 
+
+> Please, read carefully the instructions within this step. If you encounter any difficulties, go to the section [Authorizing Destinations](/components/use-cases/ecommerce/authorizing-destinations/) below.
+
+![Shopify Analytics to Keboola Provided Snowflake Database](/components/use-cases/ecommerce/shopify-to-keboola-snowflake.png)
+
+When you are finished, click **Save** in the top right corner. The template builder will create your new configuration and when it is done, you will be redirected to Use Case Catalogue and see the newly created flow. 
+
+Click :arrow_forward: **Run Use Case** and start building your visualizations a few minutes after. 
+
+![Shopify Analytics to Keboola Provided Snowflake Database - Flows](/components/use-cases/ecommerce/shopify-to-keboola-provided-snowflake-database-flows.png)
+
+## Authorizing eCommerce Data Sources
+To use a selected data source component, you must first authorize the data source. 
+
+At least one data source must be used in order to create a working flow.
+
+### Shopify Analytics
+
+![Shopify Source](/components/use-cases/ecommerce/shopify-source.png)
+
+To enable this application, you must:
+
+- [Enable private app development](https://help.shopify.com/en/manual/apps/private-apps#enable-private-app-development-from-the-shopify-admin) for your store.
+- Create a private application.
+- Enable `Read access` ADMIN API PERMISSIONS for the following following objects:
+    - `Orders`
+    - `Products`
+    - `Inventory`
+    - `Customers`
+
+Additional documentation is available [here](https://bitbucket.org/kds_consulting_team/kds-team.ex-shopify/src/master/README.md).
```

Comment:
to by bylo dobrý překlopit sem, ty linky na bitbucket jsou strašně obskurní

Reply:
@MonikaFeigler

Reply:
nerozumím.